### PR TITLE
LPS-94371 bookmarks-test: Create BookmarksFolderMultiLanguageSearchTest

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

File: modules/apps/bookmarks/bookmarks-test/src/testIntegration/java/com/liferay/bookmarks/search/test/BookmarksEntryMultiLanguageSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.rule.SynchronousDestinationTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.search.test.util.FieldValuesAssert;
+import com.liferay.portal.search.test.util.IndexerFixture;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
```
Comment (foshiro-zz) [inline modules/apps/bookmarks/bookmarks-test/src/testIntegration/java/com/liferay/bookmarks/search/test/BookmarksEntryMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

**Review by foshiro-zz — CHANGES_REQUESTED** [review]
@vagner-castro , code looks good to me. Just have to change `PermissionCheckerTestRule` to `PermissionCheckerMethodTestRule` in all occurrences

**Comment by vagnerbc** [comment]
Changes Updated

**Comment by foshiro-zz** [comment]
LGTM

**Comment by foshiro-zz** [comment]
Submitted to https://github.com/brandizzi/liferay-portal/pull/771

**Pull request closed without merging**